docs/guides/runtime/web-debugger.md

```diff
@@ -0,0 +1,82 @@
+---
+name: Debugging Bun with the web debugger
+---
+
+Bun speaks the [WebKit Inspector Protocol](https://chromedevtools.github.io/devtools-protocol/). To enable debugging when running code with Bun, use the `--inspect` flag. For demonstration purposes, consider the following simple web server.
```

Comment:
chrome devtools protocol is different

this can be a link to the typescript declarations in bun-vscode/jsc.d.ts?